merch/product.php

```diff
@@ -63,6 +63,11 @@
 	$back_file = $selected_variant->get_file_by_type(PrintfulVariantFilesTypes::BACK);
 	
 	head("Buy {$product->get_name()}", true);
+    $category_name = strtolower(preg_replace('@^.*\(([^()]+)\)$@i', '$1', $product->get_name()));

```

Comment:
there seems to be a whitespace issue here

Reply:
I love spaces vs. tab character